src/trace/Kconfig

```diff
@@ -6,9 +6,12 @@ menu "Trace"
 
 config TRACE
 	bool "Trace"
+	default n if ZEPHYR_SOF_MODULE
```

Comment:
depends on xtos ?

Reply:
@lgirdwood That would be convenient, but we don't have any Kconfig that would be unique to XTOS. There's also no common Kconfig for the XTOS DMA API, so we can't add a "depends" on that either.